pkg/sosreport/index.jsx

```diff
@@ -117,94 +267,357 @@ function sosCreate(setProgress, setError, setErrorDetail) {
+                  footer={
+                      <>
+                          {actions}
+                          {progress ? <span>{cockpit.format(_("Progress: $0"), progress.toFixed() + "%")}</span> : null}
+                      </>
+                  }
+                  title={ _("Run new report") }>
+        { error
+            ? <>
+                <Alert variant="warning" isInline title={error}>
+                    <CodeBlockCode>{errorDetail}</CodeBlockCode>
+                </Alert>
+                <br />
             </>
-        );
+            : null }
+        <p>{ _("SOS reporting collects system information to help with diagnosing problems.") }</p>
+        <p>{ _("This information is stored only on the system.") }</p>
+        <br />
+        <Form isHorizontal>
+            <FormGroup label={_("Report label")}>
+                <TextInput value={label} onChange={setLabel} />
+            </FormGroup>
+            <FormGroup label={_("Encryption passphrase")}
+                       helperText="Leave empty to skip encryption">
+                <InputGroup>
+                    <TextInput type={showPassphrase ? "text" : "password"} value={passphrase} onChange={setPassphrase}
+                               autoComplete="new-password" />
+                    <Button variant="control" onClick={() => setShowPassphrase(!showPassphrase)}>
+                        { showPassphrase ? <EyeSlashIcon /> : <EyeIcon /> }
+                    </Button>
+                </InputGroup>
+            </FormGroup>
+            <FormGroup label={_("Options")} hasNoPaddingTop>
+                <Checkbox label={_("Obfuscate network addresses, hostnames, and usernames")}
+                                 isChecked={obfuscate} onChange={setObfuscate} />
+                <Checkbox label={_("Use verbose logging")}
+                                 isChecked={verbose} onChange={setVerbose} />
+            </FormGroup>
+        </Form>
+    </Modal>;
+};
+
+const SOSRemoveDialog = ({ path }) => {
+    const D = useDialog();
+    const [task, setTask] = useState(null);
+    const [error, setError] = useState(null);
+
+    function remove() {
+        setError(null);
+        setTask(sosRemove(path)
+                .then(D.close)
+                .catch(err => {
+                    setTask(null);
+                    setError(err.toString());
+                }));
     }
 
-    return <Modal id="sos-dialog" position="top" variant="medium" isOpen onClose={onClose} actions={actions}
-                  title={ _("Create diagnostic report") }>{content}</Modal>;
+    return (
+        <Modal id="sos-remove-dialog"
+               position="top"
+               variant="medium"
+               isOpen
+               onClose={D.close}
+               title={<>
+                   <ExclamationTriangleIcon className="ct-icon-exclamation-triangle" />
+                   { "\n" }
+                   {_("Delete report permanently?")}
```

Comment:
This seem odd with the `\n`.
You should be able to do:
```
<Modal title={_("Delete report permanently?")} titleIconVariant="warning" />
```
for the same effect

Reply:
Oh, good point, I didn't know about titleIconVariant.